applications/LS-Dyna-stampede3/app.json

```diff
@@ -0,0 +1,124 @@
+                        }
+                    }
+                }
+            ],
+            "containerArgs": [],
+            "schedulerOptions": [
+                {
+                    "name": "TACC Scheduler Profile",
+                    "description": "Scheduler profile for HPC clusters at TACC",
+                    "inputMode": "FIXED",
+                    "arg": "--tapis-profile ls-dyna-2023R2",
+                    "notes": {
+                        "isHidden": true
+                    }
+                },
+                {
+                    "name": "Slurm job name",
+                    "description": "Set the slurm job name to be identical to the Tapis job name.",
+                    "inputMode": "FIXED",
+                    "arg": "--job-name ${JobName}",
+                    "notes": {
+                        "isHidden": true
+                    }
+                }
+            ],
+            "envVariables": [
+                {
+                    "key": "pre",
+                    "value": "S",
+                    "description": "Please choose the precision for the solver. S = Single Precision and D = Double Precision. Default is Single Precision.",
+                    "inputMode": "REQUIRED",
+                    "notes": {
+                      "label": "Precision",
+                      "enum_values": [
+                        {
+                          "S": "S"
+                        },
+                        {
+                          "D": "D"
+                        }
+                      ]
+                    }
+                  }
+            ],
+            "archiveFilter": {
+                "includes": [],
+                "excludes": [
+                    "tapisjob_app.sh",
+                    "ls-dyna.zip",
+                    "tapisjob.env"
+                ],
+                "includeLaunchFiles": true
+            }
+        },
+        "fileInputs": [
+            {
+                "name": "Input Directory",
+                "description": "The directory containing the files that you want to work on. This directory and its files will be copied to where your GiD session runs. Click the 'Select Input' button and then select the directory you want to work from.",
+                "inputMode": "REQUIRED",
+                "autoMountLocal": true,
+                "sourceUrl": "tapis://cloud.data/corral/tacc/aci/CEP/community/app_examples/ls-dyna",
+                "targetPath": "inputDirectory"
+            }
+        ],
+        "fileInputArrays": [],
+        "nodeCount": 1,
+        "coresPerNode": 48,
+        "memoryMB": 192000,
+        "maxMinutes": 120,
+        "subscriptions": [],
+        "tags": []
+    },
+    "tags": [
+        "portalName: DesignSafe"
+    ],
+    "notes": {
+        "icon": "LS-Dyna",
+        "label": "LS-Dyna (Stampede3)",
+        "helpUrl": "https://docs.tacc.utexas.edu/software/ansys/",
```

Comment:
```suggestion
        "helpUrl": "https://www.designsafe-ci.org/user-guide/tools/simulation/lsdyna/",
```

Reply:
I'll go ahead and update this URL. However, just a heads up, the information at [helpUrl](https://www.designsafe-ci.org/user-guide/tools/simulation/lsdyna/) seems to be a bit behind/outdated.